lib/items_repository.rb

```diff
@@ -0,0 +1,59 @@
+require 'item_model'
+
+class ItemsRepository
+  def all
+    sql = 'SELECT id, item_name, item_price, item_quantity FROM items;'
+    result = DatabaseConnection.exec_params(sql, [])
+    items_list = []
+    item = Item.new
+    loop_records(result, item, items_list)
```

Comment:
I have already commented that this is a great way to refactor the method, but maybe a comment could be helpful here to explain to the reader the purpose?